src/webgpu/api/validation/queue/destroyed/buffer.spec.ts

```diff
@@ -0,0 +1,125 @@
+export const description = `
+Tests using a destroyed buffer on a queue.
+
+TODO:
+- test renderPass/renderBundle (setVertexBuffer, setIndexBuffer)
+- test renderPass (resolveQuerySet)
+- test renderPass/computePass (setBindGroup)
+`;
+
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { ValidationTest } from '../../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('writeBuffer')
+  .desc(
+    `
+Tests that use a destroyed buffer in writeBuffer fails.
```

Comment:
nit. s/use/using/g (And in other places in this file also)

Reply:
done